care/emr/tests/test_healthcare_service_api.py

```
@@ -0,0 +1,417 @@
+            },
+        )
+
+    def create_facility_location(self, facility):
+        return baker.make(
+            "emr.FacilityLocation",
+            facility=facility,
+            name="Test Location",
+        )
+
+    def create_healthcare_service(self, facility, **kwargs):
+        from care.emr.models import HealthcareService
```

Ruff (0.12.2)

`import` should be at the top-level of a file

(PLC0415)